### ISSUE TITLE
Where can we publish this?

### ISSUE THREAD

**Issue by khinsen** [ReScience/ReScience-article#3]
Wish list (mine):
- Open access
- Open reviewing
- Low cost
- read by computational scientists from all fields

I don't think any journal satisfies all criteria.

Candidates (please suggest more!):
- [F1000Research](http://f1000research.com/)
  
  Open access, open review, and a good topic section (publishing, education, communication), but also a serious price tag (500 or 1000 US$ depending on length)
- [Computing in Science & Engineering](https://www.computer.org/web/computingnow/cise)
  
  Focused on computational science, no page charges, but no open access either
  (not even for money), nor open reviewing.
  There are two channels: peer-reviewed articles, and department contributions that are evaluated
  by the department editors. Disclaimer: I am department editor for "Scientific Programming".
- [PeerJ Computer Science](https://peerj.com/computer-science/)
  
  In spite of the name there are many articles on computational science as well. Open access
  but not open reviewing. High page charges (695 US$).

**Comment by tpoisot** [comment]
I would go for PeerJ Comp Sci. The APC is a bit high _if_ we pay for the page, not much if we pay the subscriptions (I have one already, so this might influence my judgment a bit).

**Comment by jsta** [comment]
I am not sure if this satisfies the last item on your wish-list but I recently stumbled upon:
- [Royal Society Open Science](http://rsos.royalsocietypublishing.org)

**Comment by olexandr-konovalov** [comment]
JORS ([Journal of Open Research Software](http://openresearchsoftware.metajnl.com/)) has the "Issues in Research Software" track where it "publishes full-length research papers that cover different aspects of creating, maintaining and evaluating open source research software". Such paper could also possibly fit there.

**Comment by rougier** [comment]
We still need to decide where to publish since this may impact the paper (if there is length restriction for example)

**Comment by oliviaguest** [comment]
Should we go the preprint route?

**Comment by MehdiKhamassi** [comment]
Sorry if this has already been proposed previously, but why not simply publishing it in ReScience itself? It would be like an editorial article presenting the journal. And if we spread this article to colleagues and on Internet, let's hope that it will get sufficient attention to help diffuse the message about the existence of ReScience.

**Comment by oliviaguest** [comment]
That makes me think... Do we have altmetrics for our articles?

**Comment by rougier** [comment]
@MehdiKhamassi That could be an option but I would prefer a wider audience if possible since ReScience is still quite confidential I'm afraid.

@oliviaguest No. Do you know how to do that ? What is required ?

**Comment by oliviaguest** [comment]
A DOI if I'm not mistaken.

**Comment by khinsen** [comment]
Right. Just go to the Zenodo page of any of the published ReScience paper, and activate the [Altmetrics bookmarklet](https://www.altmetric.com/products/free-tools/bookmarklet/). That shows zero whatever for the papers I tried. I suppose the problem is that the DOI cannot be traced back to the ReScience article. Tweets giving the ReScience URL don't count towards altmetrics on the Zenodo DOI.

That said, I consider altmetrics about as relevant as impact factors, so I don't really care.

**Comment by khinsen** [comment]
Publishing in ReScience: I would prefer an external platform for visibility and credibility (the latter assuming that there is some reviewing process).

Preprint publication: fine as a first step, but same problem as self-publishing in ReScience in the long run. A variant: publish on [Authorea](https://www.authorea.com/). That's more or less a preprint platform on steroids, but it is getting a lot of attention recently and we might profit from that.

**Comment by rougier** [comment]
Another potential journal (Open access, no open peer review, not specific to computer science) is [PLOS One](www.plosone.org).

**Comment by rougier** [comment]
@khinsen Is Authorea a collaborative publishing platform "only" or do they offer more ?

**Comment by khinsen** [comment]
Authorea is mainly a collaborative writing platform, but since articles can be made publicly viewable, it is also a publishing platform and this is an aspect they want to develop very actively in the future. There is a commenting system but no review of any kind, which is why I place it in the category of preprint servers.
